gpu/install_gpu_driver.sh

```diff
@@ -625,11 +627,22 @@ function install_nvidia_nccl() {
     local gcs_tarball="${pkg_bucket}/nvidia/nccl/${_shortname}/${build_tarball}"
 
     if [[ "$(hostname -s)" =~ ^test && "$(nproc)" < 32 ]] ; then
-      # do not build in tests with < 32 cores
+      # when running with fewer than 32 cores, yield to in-progress build
       sleep $(( ( RANDOM % 11 ) + 10 ))
-      while gsutil ls "${gcs_tarball}.building" 2>&1 | grep -q "${gcs_tarball}.building" ; do
-        sleep 5m
-      done
+      if gsutil ls -j "${gcs_tarball}.building" > "${local_tarball}.building.json" ; then
```

Comment:
Is `-j` a recent new feature? I couldn't find this option in my `gsutil`.

Reply:
it dumps the object metadata in JSON format ; I thought it would be `--format json` to match other gcloud commands, but I guess that it's slightly different because of gsutil.  The argument is documented for `gcloud storage ls --help` which I assumed used the same ABI as gsutil, but I don't see the argument in the `gcloud ls --help` documentation.

Reply:
as for whether it's new, I don't think it is.  I looked through the release history[1], and it seems that JSON support has been included from the beginning.  But I couldn't find the argument parsing code where the argument is detected on a quick review of [2].

[1] https://github.com/GoogleCloudPlatform/gsutil/blob/master/CHANGES.md
[2] https://github.com/GoogleCloudPlatform/gsutil